app/src/main/java/com/x8bit/bitwarden/ui/platform/components/scaffold/BitwardenScaffold.kt

```diff
@@ -52,37 +59,39 @@ fun BitwardenScaffold(
         topBar = topBar,
         bottomBar = bottomBar,
         snackbarHost = snackbarHost,
-        floatingActionButton = {
-            Box(modifier = Modifier.navigationBarsPadding()) {
-                floatingActionButton()
-            }
-        },
+        floatingActionButton = floatingActionButton,
         floatingActionButtonPosition = floatingActionButtonPosition,
         containerColor = containerColor,
         contentColor = contentColor,
-        contentWindowInsets = contentWindowInsets,
+        contentWindowInsets = WindowInsets(0.dp),
         content = { paddingValues ->
-            val internalPullToRefreshState = rememberPullToRefreshState()
-            Box(
-                modifier = Modifier.pullToRefresh(
-                    state = internalPullToRefreshState,
-                    isRefreshing = pullToRefreshState.isRefreshing,
-                    onRefresh = pullToRefreshState.onRefresh,
-                    enabled = pullToRefreshState.isEnabled,
-                ),
-            ) {
-                content(paddingValues)
-
-                PullToRefreshDefaults.Indicator(
+            Column(modifier = Modifier.padding(paddingValues)) {
+                utilityBar()
+                val internalPullToRefreshState = rememberPullToRefreshState()
+                Box(
                     modifier = Modifier
-                        .padding(paddingValues)
-                        .align(Alignment.TopCenter),
-                    isRefreshing = pullToRefreshState.isRefreshing,
-                    state = internalPullToRefreshState,
-                    containerColor = BitwardenTheme.colorScheme.background.secondary,
-                    color = BitwardenTheme.colorScheme.icon.secondary,
-                )
+                        .windowInsetsPadding(insets = contentWindowInsets)
+                        .pullToRefresh(
+                            state = internalPullToRefreshState,
+                            isRefreshing = pullToRefreshState.isRefreshing,
+                            onRefresh = pullToRefreshState.onRefresh,
+                            enabled = pullToRefreshState.isEnabled,
+                        ),
+                ) {
+                    content(PaddingValues(all = 0.dp))
```

Comment:
I wanted to get an opinion on this. Since the padding needs to be applied to the outer column, the padding no longer is needed here. I was thinking I would just remove it.

Any thoughts?